tests/tests.js

```diff
@@ -33,3 +33,21 @@ tape('Test the event details route', t => {
       });
   });
 });
+
+tape('Test the authentication middleware', t => {
+  supertest(server)
+    .get('/add-event')
+    .set('Cookie', [ 'token=eyJhbGciOiJIUzI1NiJ9.NTdhYjM2NTQ0NGUxYjhhOTY3NzExMWMzZjYxY2VjNzFiNzMyNjZhYw.Fpzao_FN05j29auCfrrM0-eNhqYJQ_QYbzJiXnZXSQA' ])
```

Comment:
you shouldn't hard code this, use `jwt.sign`

Reply:
^^

Reply:
@Karyum

Reply:
@mattlub sorry about didn't see it i was trying to fix travis lol

Reply:
@mattlub :shipit: